tests/suites/test_suite_bignum_mod.function

```diff
@@ -17,20 +17,41 @@ void mpi_mod_setup( int ext_rep, int int_rep, int iret )
     #define MLIMBS 8
     mbedtls_mpi_uint mp[MLIMBS];
     mbedtls_mpi_mod_modulus m;
+    const size_t mp_size = sizeof(mbedtls_mpi_uint);
     int ret;
 
-    memset( mp, 0xFF, sizeof(mp) );
+    memset( mp, 0xFF, mp_size );
```

Comment:
`mp_size` is not the number of bytes in `mp` (which is what `sizeof(mp)` is: it's the number of bytes in a single `mbedtls_mpi_uint`. Is it intentional to only set the first 4 or 8 bytes here?